react-counter/src/components/Counter.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSync } from '@fortawesome/free-solid-svg-icons';
+
+const element = <FontAwesomeIcon icon={faSync} />;
+
+class Counter extends React.Component{
+    constructor(props){
+        super(props);
+
+        this.state = {
+            value: 0,
+        }
+    }
+
+    plus = () => {
+        this.setState({
+            value: this.state.value + 1,
+        })
```

Comment:
when changing the state to access the previous state value you should use a callback instead:
```suggestion
        this.setState((prevState) => ({
            value: prevState.value + 1,
        }))
```
check out the docs page: https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous